products/compute/inspec.yaml

```diff
@@ -37,22 +49,42 @@ overrides: !ruby/object:Overrides::ResourceOverrides
   Image: !ruby/object:Overrides::Inspec::ResourceOverride
     exclude: true
   Instance: !ruby/object:Overrides::Inspec::ResourceOverride
-    exclude: true
+    additional_functions: |
```

Comment:
What do you think about additional_functions taking in a template name like singular_extra_examples?